source/extensions/common/wasm/ext/set_envoy_filter_state.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package envoy.source.extensions.common.wasm;
+
+message SetEnvoyFilterStateArguments {
+  string path = 1;
```

Comment:
Please add comments here, path is the filter state key, value is the factory input.

Reply:
done